stylesheets/_component.datatables.scss

```diff
@@ -47,59 +47,76 @@
 // scss-lint:enable SelectorFormat
 
 a.dt-button {
-    border-right: 1px solid color(border);
+    border-right: 1px dotted color(border);
     margin-right: $margin-base-vertical;
     padding-right: $margin-base-vertical;
 }
 
-.dt-buttons > a,
-.dt-buttons > button {
+.dt-buttons {
     float: left;
-    margin-top: $margin-large-vertical;
-}
-
-.dt-buttons > button {
-    background-color: transparent;
-    border: 0;
-    border-right: 1px solid color(border);
-    color: color(jadu-blue, dark);
-    margin-right: $margin-base-vertical;
-    padding: 0 $padding-base-horizontal;
 
-    &:hover {
-        color: color(jadu-blue, darkest);
+    > button {
+        background-color: transparent;
+        border: 0;
+        border-right: 1px dotted color(border);
+        color: color(link);
+        float: left;
+        line-height: 30px;
+        padding: 0 $padding-base-horizontal;
+
+        &:hover {
+            color: color(link, hover);
+        }
     }
 }
 
 .datatable {
     // scss-lint:disable SelectorFormat
     td,
     th {
+        &.sorting:hover,
+        &.sorting_asc:hover,
+        &.sorting_desc:hover {
+            background-color: color(background, dark);
+            cursor: pointer;
+
+            &::after {
+                color: color(link);
+            }
+        }
+
         &.sorting::after,
         &.sorting_asc::after,
         &.sorting_desc::after {
             @include icon-FontAwesome();
-            font-size: 18px;
+            font-size: 16px;
             left: 0;
             margin-left: 10px;
         }
 
+        &.sorting::after {
```

Comment:
We need a `not(.sorting_disabled)` in there to prevent the icon being added when sorting is disabled

Reply:
Good catch, added